transports/noise/Cargo.toml

```diff
@@ -23,6 +23,7 @@ static_assertions = "1"
 thiserror = "1.0.40"
 x25519-dalek = "1.1.0"
 zeroize = "1"
+bs58 = "0.5.0"
```

Comment:
If I understand correctly, this dependency is only needed to display `UnknownWebTransportCerthashes`. Though we already depend on it in `libp2p-identity` and I don't see that usage going away any time soon. Thus in my eyes, this is fine.

Reply:
I removed `bs58` because now I wrap the hash with `Protocol::Certhash`